package.json

```diff
@@ -4,7 +4,9 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "node index.js"
+    "start": "node src/index.js",
+    "start-dev": "NODE_ENV=development ./node_modules/.bin/nodemon src/index.js",
+    "pretty": "./node_modules/.bin/prettier --loglevel warn --write src"
```

Comment:
Rules can be configured as needed. I just threw in what seemed to be existing convention (single quotes, no semi).